telebot/types.py

```diff
@@ -1274,10 +1280,19 @@ def __init__(self, message_id, from_user, date, chat, content_type, options, jso
         self.write_access_allowed: Optional[WriteAccessAllowed] = None
         self.user_shared: Optional[UserShared] = None
         self.chat_shared: Optional[ChatShared] = None
+        self.story: Optional[Story] = None
         for key in options:
             setattr(self, key, options[key])
         self.json = json_string
 
+    @property
+    def is_forwarded_story(self):
```

Comment:
See no reason in appearance of this property... No is_anumation, is_video, is_xxx - why now is_story?